pkg/security/ebpf/c/include/maps.h

```diff
@@ -41,7 +41,8 @@ BPF_HASH_MAP(secprofs_syscalls, u64, struct security_profile_syscalls_t, 1) // m
 BPF_HASH_MAP(auid_approvers, u32, struct event_mask_filter_t, 128)
 BPF_HASH_MAP(auid_range_approvers, u32, struct u32_range_filter_t, EVENT_MAX)
 
-BPF_LRU_MAP(activity_dump_rate_limiters, u64, struct activity_dump_rate_limiter_ctx, 1) // max entries will be overridden at runtime
+BPF_LRU_MAP(activity_dump_rate_limiters, u64, struct rate_limiter_ctx, 1) // max entries will be overridden at runtime
+BPF_LRU_MAP(rate_limiters, u32, struct rate_limiter_ctx, 1) // max entries will be overridden at runtime
```

Comment:
see my comment about a common map and function

Reply:
I've think about it and my take is that: as the use case are a bit different, IMHO it make sense to have them separated:
* a first one with a 64bit cookie, sized to MaxTracedCgroupsCount
* a second one with a pid 32bit key, sized as the other pid caches
This way, we ensure entries for AD are not evicted on process stressed activity.
Also, the rates should be different (number of syscalls done by a whole container VS number of syscalls for a single process).
WDYT?